### PR TITLE
Feat/site banner

### PR DESCRIPTION
**Pull request by jomarmontuya**

### CODE REVIEW

**Comment by jomarmontuya** [comment]
image

**Review by ardeay — CHANGES_REQUESTED** [review]
This need a dismiss button. That dismiss button will set a cookie, that cookie is checked before the banner loads and choses to show or not show the banner.

**Comment by jomarmontuya** [comment]
image

image


Session cookie is set on banner dismissed

File: src/components/marketing/SiteBanner/SiteBanner.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                  component: Typography,
+                  props: {
+                    component: 'a',
+                    variant: 'body1',
+                    color: theme.palette.common.white,
+                  },
+                },
+              }}
+            >
+              {bannerContent.data[0].content.banner_content ||
```
Comment (darwin808) [inline src/components/marketing/SiteBanner/SiteBanner.js]: can we add a checker that bannerContent is Array to prevent error thanks

**Comment by jomarmontuya** [comment]
Dev Branch Preview -> https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/

Dev branch is sync with current stage branch

**Comment by jomarmontuya** [comment]
https://user-images.githubusercontent.com/61284357/196968475-d2798bad-66ee-436e-849a-6bb3ca3948f1.mp4

**Review by darwin808 — APPROVED** [review]

**Pull request merged**